braviarc/braviarc.py

```diff
@@ -98,7 +98,7 @@ def _wakeonlan(self):
 
     def send_req_ircc(self, params, log_errors=True):
         """Send an IRCC command via HTTP to Sony Bravia."""
-        headers = {'SOAPACTION': 'urn:schemas-sony-com:service:IRCC:1#X_SendIRCC'}
+        headers = {'SOAPACTION': '"urn:schemas-sony-com:service:IRCC:1#X_SendIRCC"'}
```

Comment:
On the android tvs if you don't enclose the SOAPACTION header in double quotes the IRCC commands don't work. You get a 401 error from the upnp service on the tv.